SeaBattleTest/Ships.cs

```diff
@@ -0,0 +1,115 @@
+﻿using SeaBattle;
+using System;
+using System.Collections.Generic;
+using SeaBattle;
```

Comment:
Double using SeaBattle and in general a lot of unused imports, keep code clean

Reply:
This is a good question, a lot of using appear themselves maybe after start testing or how it can happen every time? I typed using SeaBattle and FluentAssertion only by myself. Cleared usings.

Reply:
Idk why it happening to you, but you could at least check it before doing the commit.